src/templates/Article/index.js

```diff
@@ -514,6 +523,35 @@ const createSchema = ({
                 },
                 isVoid: true
               },
+              {
+                matchMdast: matchZone('EMBEDCOMMENT'),
+                props: node => ({ nodeData: node.data }),
+                component: withCommentData(({ nodeData, liveData }) => {
+                  console.log(nodeData, liveData)
+                  if (!liveData) return null
+                  return (
+                    <Loader
+                      error={liveData.error}
+                      loading={liveData.loading}
+                      render={() => {
+                        return (
+                          <TeaserEmbedComment
+                            nodeData={nodeData}
+                            liveData={liveData}
+                            t={t}
+                            Link={CommentLink}
+                          />
+                        )
```

Comment:
@tpreusse this part still doesn't handle well the case where an id doesn't exist. The error gets returned from the BE, but is subsequently "erased", which causes all sorts of trouble, but i'mnot able to pinpoint where. Happens both for FE and Publikator. Any advice welcome :)